### PR TITLE
PBM-665 Transition docs to Netlify 

### PR DESCRIPTION
**Pull request by nastena1606**
Settings to deploy documentation on Netlify platform

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 4.0 | passed |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 3.6 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Sharded cluster mongodb 4.0 | passed |
| Non-sharded replicaset mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 4.0 | passed |
| Sharded cluster mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.0 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Sharded cluster mongodb 4.0 | passed |
| Sharded cluster mongodb 3.6 | passed |
| Non-sharded replicaset mongodb 3.6 | passed |
| Sharded cluster mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 4.0 | passed |
| Non-sharded replicaset mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 4.0 | passed |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 4.0 | passed |
| Non-sharded replicaset mongodb 3.6 | passed |
| Non-sharded replicaset mongodb 4.2 | passed |
| Sharded cluster mongodb 4.2 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Sharded cluster mongodb 4.0 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Sharded cluster mongodb 4.0 | failed |
| Restore on new cluster mongodb 4.0 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Restore on new cluster mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 3.6 | passed |
| Sharded cluster mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 4.0 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.0 | passed |
| Non-sharded replicaset mongodb 3.6 | passed |
| Non-sharded replicaset mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 4.0 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Sharded cluster mongodb 4.0 | passed |
| Sharded cluster mongodb 3.6 | passed |
| Sharded cluster mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 4.2 | passed |
| Restore on new cluster mongodb 4.0 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Restore on new cluster mongodb 4.0 | passed |
| Sharded cluster mongodb 4.0 | passed |
| Sharded cluster mongodb 4.2 | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Sharded cluster mongodb 4.2 | passed |
| Sharded cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 3.6 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Sharded cluster mongodb 3.6 | failed |
| Sharded cluster mongodb 4.0 | failed |
| Sharded cluster mongodb 4.2 | failed |
| Restore on new cluster mongodb 3.6 | passed |
| Non-sharded replicaset mongodb 3.6 | failed |
| Restore on new cluster mongodb 4.0 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Non-sharded replicaset mongodb 4.0 | failed |
| Single-node replicaset mongodb 3.6 | failed |
| Non-sharded replicaset mongodb 4.2 | failed |
| Single-node replicaset mongodb 4.0 | failed |
| Single-node replicaset mongodb 4.2 | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Restore on new cluster mongodb 4.0 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Sharded cluster mongodb 4.2 | failed |
| Sharded cluster mongodb 4.0 | failed |
| Non-sharded replicaset mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 3.6 | failed |
| Non-sharded replicaset mongodb 4.0 | failed |
| Single-node replicaset mongodb 3.6 | failed |
| Single-node replicaset mongodb 4.2 | failed |
| Single-node replicaset mongodb 4.0 | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Sharded cluster mongodb 4.0 | failed |
| Sharded cluster mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 3.6 | failed |
| Non-sharded replicaset mongodb 4.0 | failed |
| Restore on new cluster mongodb 4.0 | passed |
| Single-node replicaset mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 4.2 | failed |
| Single-node replicaset mongodb 4.0 | failed |
| Single-node replicaset mongodb 3.6 | failed |
| Restore on new cluster mongodb 4.2 | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Sharded cluster mongodb 3.6 | failed |
| Sharded cluster mongodb 4.0 | failed |
| Sharded cluster mongodb 4.2 | failed |
| Single-node replicaset mongodb 3.6 | failed |
| Restore on new cluster mongodb 4.0 | passed |
| Non-sharded replicaset mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 4.0 | failed |
| Single-node replicaset mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 3.6 | failed |
| Single-node replicaset mongodb 4.0 | failed |
| Restore on new cluster mongodb 4.2 | passed |

File: doc/.netlify/state.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (PaulJacobs-percona) [inline doc/.netlify/state.json:1]: What does this file do?

Comment (nastena1606) [reply doc/.netlify/state.json:1]: This is a link to a local site from my machine to the one on Netlify. It was created automatically when i deployed the docs manually from netlify CLI

**Review by PaulJacobs-percona — APPROVED** [review]

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**